src/ravi_jitshared.c

```diff
@@ -299,19 +299,6 @@ static const char Lua_header[] =
     "	{ TValue *io=(o); const Udata *iu = (u); \\\n"
     "	  io->value_ = iu->user_; settt_(io, iu->ttuv_); \\\n"
     "	  checkliveness(L,io); }\n"
-    "typedef enum {\n"
-    "	RAVI_TANY = 0,\n"
-    "	RAVI_TNUMINT = 1,\n"
-    "	RAVI_TNUMFLT,\n"
-    "	RAVI_TARRAYINT,\n"
-    "	RAVI_TARRAYFLT,\n"
-    "	RAVI_TFUNCTION,\n"
-    "	RAVI_TTABLE,\n"
-    "	RAVI_TSTRING,\n"
-    "	RAVI_TNIL,\n"
-    "	RAVI_TBOOLEAN,\n"
-    "	RAVI_TUSERDATA\n"
-    "} ravitype_t;\n"
     "typedef struct Upvaldesc {\n"
```

Comment:
we should also remove the struct definitions of Upvaldesc and LocVar where ravi_type appears

Reply:
No, these might be used to look up things in the JIT. I should add `ravi_type_map` and adjust the fields.